StaticAnalyzer/views/android.py

```diff
@@ -606,6 +621,10 @@ def StaticAnalyzer(request):
                                                                                      ANDROVER=ANDROVER,
                                                                                      MANIFEST_ANAL=MANIFEST_ANAL,
                                                                                      PERMISSIONS=PERMISSIONS,
+
+# Esteve 21.08.2016 - begin - Permission Analysis with Androguard
+                                                                                     ANDROPERMS = "",
```

Comment:
Hi Ajin. The reason for this would be the same as the reason in the previous review comment. The androlyze tool of the Androguard framework would not be able to receive a ZIP as input.